commit 14aced4020ecd381418aa5703bb0b0a4ea68d022
Author: Paul Hüber <phueber@kernsp.in>
Date:   Sun Feb 26 17:58:19 2017 +0100

    l2tp: avoid use-after-free caused by l2tp_ip_backlog_recv
    
    [ Upstream commit 51fb60eb162ab84c5edf2ae9c63cf0b878e5547e ]
    
    l2tp_ip_backlog_recv may not return -1 if the packet gets dropped.
    The return value is passed up to ip_local_deliver_finish, which treats
    negative values as an IP protocol number for resubmission.
    
    Signed-off-by: Paul Hüber <phueber@kernsp.in>
    Signed-off-by: David S. Miller <davem@davemloft.net>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1379_14aced4020ecd381418aa5703bb0b0a4ea68d022.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1379_14aced4020ecd381418aa5703bb0b0a4ea68d022.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
